### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by selvanair**
- Update the description of registry keys
- Document --command cmd [arg]   (relevant only after merging PR 188)

### CODE REVIEW

**Comment by selvanair** [comment]
@mattock : does this look ok?

File: README.rst [hunk 1]
```diff
@@ -172,10 +207,18 @@ disable_save_passwords
     Set to a nonzero value to disable the password save feature.
     Default: 0
 
+User Preferences
+****************
+
 All other OpenVPN GUI registry values are located below the
-*HKEY_CURRENT_USER\\SOFTWARE\\OpenVPN-GUI\\* key
+*HKEY_CURRENT_USER\\SOFTWARE\\OpenVPN-GUI\\* key. In a fresh
+instalaltion none of these values are present and are not
```
Comment (mattock) [inline README.rst]: Typo: instalaltion -> installation

**Comment by mattock** [comment]
ACK once the typo is fixed.

File: README.rst (hunk 1 above)
Comment (selvanair) [reply README.rst]: Thanks. Typo fixed; merging

**Pull request merged**